src/store/albums.js

```diff
@@ -44,21 +44,19 @@ export default {
         }
       }
     },
-    updateLabelOccurence(state, { newID, oldID }) {
-      for (const a in state.albums) {
-        const i = state.albums[a].label_ids.findIndex(
-          (lId) => `${lId}` === `${oldID}`
-        );
+    updateLabelOccurence(state, { oldID, newID }) {
+      const oldAlbums = state.albums;
```

Comment:
Is there a reason why this doesn't use the spread operator? I would want to keep all the commit as similar as possible

Reply:
If it isn't necessary to use a spread operator, we should remove it for all `const oldObjects = `

Reply:
Because it's not necessary? No need to create a new (large) object if it's not required.

Reply:
> If it isn't necessary to use a spread operator, we should remove it for all `const oldObjects = `

Sure, agreed.